src/interactions/hoverInteraction.ts

```diff
@@ -45,14 +46,20 @@ export module Interaction {
 
     public _anchor(component: Hoverable, hitBox: D3.Selection) {
       super._anchor(component, hitBox);
-      this._dispatcher = new Dispatcher.Mouse(this._hitBox);
+      this._dispatcher = Dispatcher.Mouse.getDispatcher(<SVGElement> (<any> this._componentToListenTo)._element.node());
```

Comment:
Would you still want the `_element` to be an API point (aside)?  I remember you saying that it would be useful.